src/tests/ftest/util/telemetry_utils.py

```diff
@@ -147,7 +147,8 @@ class TelemetryUtils():
     ENGINE_POOL_VOS_WAL_METRICS = [
         *_gen_stats_metrics("engine_pool_vos_wal_wal_sz"),
         *_gen_stats_metrics("engine_pool_vos_wal_wal_qd"),
-        *_gen_stats_metrics("engine_pool_vos_wal_wal_waiters")]
+        *_gen_stats_metrics("engine_pool_vos_wal_wal_waiters"),
+        *_gen_stats_metrics("engine_pool_vos_wal_wal_lat")]
```

Comment:
Would you update the WAL metrics test in src/tests/ftests/telemetry/wal_metrics.py?  I think only the test_wal_commit_metrics() needs be updated.

Reply:
- [ ] Update `telemetry/wal_metrics.py` functional test

Reply:
> * [x]  Update `telemetry/wal_metrics.py` functional test

Fixed with commit 4cc65e7ee